tests/test1.yaml

```
@@ -1706,6 +1720,14 @@ sensor:
       memory_location: 0x20
       memory_address: 0x7d
       name: Adres sensor
+  - platform: uponor_smatrix
+    address: 0xDE13
+    humidity:
+      name: "Thermostat Humidity Living Room"
+    temperature:
+      name: "Thermostat Temperature Living Room"
+    external_temperature:
+      name: "Thermostat Floor Temperature Living Room"
```

Quotes not required in simple strings in YAML
```suggestion
      name: Thermostat Humidity Living Room
    temperature:
      name: Thermostat Temperature Living Room
    external_temperature:
      name: Thermostat Floor Temperature Living Room
```